docs/DeveloperGuide.md

```diff
@@ -58,6 +58,16 @@ the following packages.
 * Task: Contains the classes for the different types of tasks.
 * Ui: In charge of interaction with the user.
 
+####SubjectCommand
+![AddSubjectSequenceDiagram](https://user-images.githubusercontent.com/47527482/97392568-0bf0ab80-191d-11eb-99cf-c98d7863ba91.png)
```

Comment:
![image](https://user-images.githubusercontent.com/15065550/97544604-d1fad480-1a04-11eb-8d92-d47a2f4ba03f.png)
![image](https://user-images.githubusercontent.com/15065550/97544717-f656b100-1a04-11eb-8f19-b6437b301776.png)
The end of the activation bar looks kinda incomplete for both diagrams. In the second image the parse() call to TopicParser seems incomplete as well.